### ISSUE TITLE
Possible CPTAC Bug - See link

### ISSUE THREAD

**Issue by jjacobson95** [PNNL-CompBio/coderdata#87]
I'll run again shortly and comment the result. Sometimes connectivity or site instability can cause script failures. If this is the case, a longer term solution would be to check get/post response and rerun within the script if there is a failure.

https://github.com/PNNL-CompBio/coderdata/actions/runs/7700637854/job/20986560790#step:9:21

**Comment by sgosline** [comment]
I dont understand this error. Can you give me information on how to repro?

**Comment by jjacobson95** [comment]
As no changes have been made to the CPTAC script, I assume there are two possible options for this failure:
1) a site connectivity issue - so if this script fails to receive data from the site, it proceeds until it hits a error resulting from this. This may be avoided by checking the message associated the data pull or the format of the data and rerunning if it fails to retrieve desired data.
2) CPTAC made an update so that the data thats being pulled is in a different format and is causing some failure.

I'm currently trying to reproduce this using the CI pipeline. If it fails again, I'll check if it is reproducible via docker (and is likely the second issue). If it doesn't fail, it is likely the first issue and is a lower priority bug.

Either way, I'll share the result

**Comment by sgosline** [comment]
Ok, i still can't repro without more details. Let me know what branch or environment and what command it fails on and I can try to help.

**Comment by sgosline** [comment]
the cptac python package is broken. Filed issue here: https://github.com/PayneLab/cptac/issues/60

**Comment by jjacobson95** [comment]
Currently failing on Main and JJ builder branch. 

The command its failing on is as follows during the sample file generation step.
docker run -v $PWD:/tmp cptac-samples-builder --geneFile=genes.csv --prevSampleFile=/tmp/cell_line_samples.csv

**Comment by jjacobson95** [comment]
Ah, that would do it.

**Comment by sgosline** [comment]
to repro fully it requires a build first.
```
docker build -f dockerfile.cptac  -t cptac . 
docker run -v $PWD:/tmp cptac --geneFile=genes.csv --prevSampleFile=/tmp/cell_line_samples.csv 
```

**Comment by jjacobson95** [comment]
Fixed by setting version of cptac install to 1.5.8.
